spec/support/medium_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
+include Rails.application.routes.url_helpers
+
+RSpec.shared_examples "a medium controller" do
+
+  describe "GET 'index'" do
+    it "is successful" do
+      get :index
+      expect(response.status).to eq 200
+    end
+  end
+
+  describe "GET 'new'" do
+    it "renders new view" do
+      get :new
+      expect(subject).to render_template :new
+    end
+  end
+
+  describe "POST 'create'" do
+    before :each do
+      post :create, good_params
+    end
+
+    it "redirects to show page" do
+      expect(subject).to redirect_to polymorphic_path(assigns(media_type))
+    end
+
+    it "renders new template on error" do
+      post :create, bad_params
+      expect(subject).to render_template :new
+    end
```

Comment:
Like you noted, it's still a good idea to test the basic model actions of create/update/destroy. If it was a test specific to the Albums model, for instance (and not a shared example), it might look something like this: 
```
it "creates a record" do
    post :create, album_params

    expect(Album.count).to eq 1 
end
```
And a creation test for invalid params `to eq 0`, etc.